samri/pipelines/nipype_based/diagnostics.py

```diff
@@ -149,15 +150,25 @@ def diagnose(measurements_base,
 			(s_bru2nii, datasink, [('nii_file', 'anat')]),
 			])
 
+	#TODO: realigner  optional / als flag (wenn spm) ansonsten mandatory, slicetiming nicht spm
+	#TODO: realign vs. not realign, ICA geht verloren auf neurostars posten
```

Comment:
Please translate to english and add link to forum thread https://neurostars.org/t/nipy-spacetimerealigner-problem/337/7